opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -76,7 +76,7 @@ message ScopeSpans {
 
 // A Span represents a single operation performed by a single component of the system.
 //
-// The next available field id is 17.
+// The next available field id is 18.
```

Comment:
In the existing Span structure, I can't seem to find a field with id 16. If field id 16 is not being used, should we use field id 16 for this newly added field?

Reply:
Yeah the comment said I should use 17 so I did, but I agree that 16 looks available and is not declared `reserved`.

Reply:
Thanks @jmacd. In the latest iteration of this PR, I see it is " fixed32 flags = 18;", should this be using field id 16?